templates/english/topic-page.html

```diff
@@ -0,0 +1,475 @@
+        <gcds-breadcrumbs-item href="#">Link</gcds-breadcrumbs-item>
+      </gcds-breadcrumbs>
+    </gcds-header>
+
+    <!---------- Main content ---------->
+    <!-- TODO: Confirm the id matches the "skip-to-href" id in the header. -->
+    <gcds-container
+      id="main-content"
+      size="full"
+      tag="main"
+    >
+      <gcds-container
+        size="xl"
+        main-container
+        centered
+        tag="div"
+      >
+        <!-- TODO: Update the heading, image and text content. -->
+        <gcds-grid
+          tag="div"
+          columns-desktop="1fr 1fr"
+          columns-tablet="1fr"
+        >
+          <div>
+            <gcds-heading tag="h1">Topic title</gcds-heading>
+            <gcds-text>
+              1-2 sentences that describe the topics and top tasks that can be accessed on this page.
+            </gcds-text>
+          </div>
+          <div class="d-none lg:d-block">
+            <img src="https://dummyimage.com/520x200/000000/FFFFFF.png" alt="" class="">
+          </div>
+        </gcds-grid>
+      </gcds-container>
+
+      <!---------- Most requested ---------->
+      <!-- TODO: Update the links href and text content. -->
+      <section class="bg-light pt-400 pb-400 mb-400">
+        <gcds-container
+          id="main-content"
+          size="xl"
+          centered
+          main-container
+          tag="div"
+        >
+          <gcds-grid
+            tag="div"
+            columns-desktop="1fr 2fr"
+          >
+            <gcds-heading margin-top="0" tag="h2">Most requested</gcds-heading>
+            <gcds-grid
+              tag="ul"
+              columns-desktop="1fr 1fr"
+              equal-row-height
+              class="ps-450 lg:ps-0"
+            >
+              <li>
+                <gcds-link href="#">
+                  Link
+                </gcds-link>
+              </li>
+              <li>
+                <gcds-link href="#">
+                  Link
+                </gcds-link>
+              </li>
+              <li>
+                <gcds-link href="#">
+                  Link
+                </gcds-link>
+              </li>
+              <li>
+                <gcds-link href="#">
+                  Link
+                </gcds-link>
+              </li>
+              <li>
+                <gcds-link href="#">
+                  Link
+                </gcds-link>
+              </li>
+              <li>
+                <gcds-link href="#">
+                  Link
+                </gcds-link>
+              </li>
+            </gcds-grid>
+          </gcds-grid>
+        </gcds-container>
+      </section>
+
+      <gcds-container
+        size="xl"
+        main-container
+        centered
+        tag="div"
+        class="pb-300"
+      >
+
+        <!---------- Services and information ---------->
+        <!-- TODO: Update the heading and text content. -->
+        <section>
+          <gcds-sr-only>
```

Comment:
The `gcds-sr-only` component has a tag prop that can be set to `h2`. Would it be possible to use that here instead of adding the `gcds-heading`?